app/Enums/ConditionName.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Otoszroto\Enums;
+
+enum ConditionName: string
```

Comment:
```suggestion
enum Condition: string
```

Jak wyżej, dobra nazwa się zwolniła.